build_dep.sh

```diff
@@ -0,0 +1,239 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+#
+# step 1: ./build_dep.sh <C|U>   C: the user in China, U: the user in US
+# step 2: source ~/.bashrc
+#
+
+DIR=/tmp/download
+mkdir $DIR
+trap "rm -fr $DIR" EXIT
+
+url_addr=https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/build-deb
+if [[ $1 != C ]] && [[ $1 != U ]]; then
+    echo "Usage: ${0} <C|U>  # C: the user in China, U: the user in US"
+    exit -1
+fi
+
+[[ $1 == U ]] && url_addr=https://nebula-graph-us.oss-us-west-1.aliyuncs.com/build-deb/
+
+# fedora
+function fedora_install {
+    echo "###### start install dep in fedora ######"
+    sudo yum -y install autoconf \
+        autoconf-archive \
+        automake \
+        bison \
+        bzip2-devel \
+        cmake \
+        curl \
+        gcc \
+        gcc-c++ \
+        java-1.8.0-openjdk \
+        java-1.8.0-openjdk-devel \
+        libstdc++-static \
+        libstdc++-devel \
+        libtool \
+        make \
+        maven \
+        ncurses \
+        ncurses-devel \
+        perl \
+        perl-WWW-Curl \
+        python \
+        readline \
+        readline-devel \
+        unzip \
+        xz-devel \
+        wget
+    wget $url_addr/fedora.tar.gz
+    tar xf fedora.tar.gz
+    pushd fedora
+        sudo rpm -ivh *.rpm
+    popd
+    wget $url_addr/vs-nebula-3rdparty.fc.x86_64.rpm
+    sudo rpm -ivh vs-nebula-3rdparty.fc.x86_64.rpm
+    echo "alias cmake='cmake -DNEBULA_GPERF_BIN_DIR=/opt/nebula/gperf/bin -DNEBULA_FLEX_ROOT=/opt/nebula/flex -DNEBULA_BOOST_ROOT=/opt/nebula/boost -DNEBULA_OPENSSL_ROOT=/opt/nebula/openssl -DNEBULA_KRB5_ROOT=/opt/nebula/krb5 -DNEBULA_LIBUNWIND_ROOT=/opt/nebula/libunwind ..'" >> ~/.bashrc
```

Comment:
Is the `..` directory in the end of the alias command needed ?

Reply:
Why not update `PATH` env variable to replace `alias cmake`?

Reply:
has cmake args